### PR TITLE
Update buidler

### PR DESCRIPTION
**Pull request by novaknole**
https://linear.app/aragon/issue/ARA2-61/govern-token-update-buidler-to-hardhat

### CODE REVIEW

File: packages/govern-token/deploy/factory.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { print } = require('../lib/utils')
+
```
Comment (nivida) [inline packages/govern-token/deploy/factory.ts]: So many breaks aren't required.

File: packages/govern-token/hardhat.config.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { config as dotenvConfig } from 'dotenv'
+import { resolve } from 'path'
+dotenvConfig({ path: resolve(__dirname, './.env') })
```
Comment (nivida) [inline packages/govern-token/hardhat.config.ts]: Could you not also have it after the imports?

File: packages/govern-token/lib/utils.js [hunk 3]
```diff
@@ -5,4 +5,4 @@ const format = ({ address }, name) =>
 
 const print = (contract, name) => console.log(format(contract, name))
 
-module.exports = { format, print }
+module.exports = { format, print }
```
Comment (nivida) [inline packages/govern-token/lib/utils.js]: EOF

**Review by nivida — CHANGES_REQUESTED** [review]

File: packages/govern-token/deploy/factory.ts (hunk 1 above)
Comment (novaknole) [reply packages/govern-token/deploy/factory.ts]: Missed this. Thanks ^__^

**Pull request merged**